Directory.Build.props

```diff
@@ -57,7 +58,7 @@
     </None>
   </ItemGroup>
 
-  <ItemGroup Label="Build Tools" Condition="$([MSBuild]::IsOsPlatform('Windows')) and '$(TargetFramework)' != 'net8.0'">
+  <ItemGroup Label="Build Tools" Condition="$([MSBuild]::IsOsPlatform('Windows')) and '$(TargetFrameworkIdentifier)' != 'NET8_0_OR_GREATER'">
```

Comment:
this doesn't seem to work as we would think it would. I would target each specifically for now.

Reply:
Reverted. Thanks Jenny!